jobs/platform/kops_gce.env

```diff
@@ -0,0 +1,20 @@
+# TODO(zmerlynn): Eliminate the other uses of this env variable
+KUBE_SSH_USER=admin
```

Comment:
I suspect I'm going to have to tweak these options, but hopefully this will only affect log dumping